### ISSUE TITLE
Failed to remove the default collection. 

### ISSUE THREAD

**Issue by zchdu** [GaiaNet-AI/gaianet-node#56]
When I execute the command  "gaianet init"
It show :

[+] Checking the config.json file ...

[+] Downloading Phi-3-mini-4k-instruct-Q5_K_M.gguf ...
    * Using the cached Phi-3-mini-4k-instruct-Q5_K_M.gguf in /home/shy/gaianet

[+] Downloading all-MiniLM-L6-v2-ggml-model-f16.gguf ...
    * Using the cached all-MiniLM-L6-v2-ggml-model-f16.gguf in /home/shy/gaianet

[+] Creating 'default' collection in the Qdrant instance ...
    * Start a Qdrant instance ...

    * Remove the existed 'default' Qdrant collection ...

      Failed to remove the default collection.

**Comment by franksprotocols** [comment]
could firstly gaianet stop, and check your task manager to see if qdrant still live in the tasklist, or you could stop it. then Gainet init

**Comment by franksprotocols** [comment]
if doesn't work, you can search GAIANET folder, and delete the whole folder, install by  running 
curl -sSfL 'https://raw.githubusercontent.com/GaiaNet-AI/gaianet-node/main/install.sh' | bash
once again.

**Comment by zchdu** [comment]
Thank you for your reply, in fact the reason for this problem is that I use a proxy, and my proxy can not access localhost. 
The solution is  follows:
In the gaianet script, when using curl to access the local galanet, add the --noproxy localhost option

These are the two places that need to be modified.
https://github.com/GaiaNet-AI/gaianet-node/blob/95f8feecfae4f3069631ef609a0ea858bafb9c40/gaianet#L211

del_response=$(curl --noproxy localhost -s -X DELETE http://localhost:6333/collections/$embedding_collection_name \
-H "Content-Type: application/json")

https://github.com/GaiaNet-AI/gaianet-node/blob/95f8feecfae4f3069631ef609a0ea858bafb9c40/gaianet#L254

response=$(curl --noproxy localhost -s -X POST http://localhost:6333/collections/$embedding_collection_name/snapshots/upload?priority=snapshot \
-H 'Content-Type:multipart/form-data' \
-F 'snapshot=@default.snapshot')

**Comment by zchdu** [comment]
如果有人遇到和我一样的问题（我想大部分是在中国），请使用上述的解决方法。

**Comment by zchdu** [comment]
another place need to be modified:
https://github.com/GaiaNet-AI/gaianet-node/blob/95f8feecfae4f3069631ef609a0ea858bafb9c40/gaianet#L778

**Comment by darktuJJ** [comment]
> 公司遇到提到同样的问题（我想大部分在国内），请使用上述的解决方案。

按照您的方法修改了211和254行，还是出现同样的问题 @zchdu

**Comment by juntao** [comment]
If you are using shadowsockets or another VPN, you could try this in the terminal

```
export no_proxy=localhost,127.0.0.0/8
```

**Comment by darktuJJ** [comment]
> If you are using shadowsockets or another VPN, you could try this in the terminal
> 
> ```
> export no_proxy=localhost,127.0.0.0/8
> ```

thanks,I will try it

**Comment by JYC0413** [comment]
> If you are using shadowsockets or another VPN, you could try this in the terminal
> 
> ```
> export no_proxy=localhost,127.0.0.0/8
> ```

this works for me！

I got this error. At first I tried to delete the qdrant folder in gaianet, but I got a new error `[Error] Failed to recover from the collection snapshot. An error occurred processing field `snapshot`: File I/O error : No such file or directory (os error 2)`. So I found this issues, enter this command and delete other files in gaianet except *.gguf (this operation can save a lot of time)

Finally I successfully started the service！！！
![image](https://github.com/GaiaNet-AI/gaianet-node/assets/71171646/91bcded7-0781-4617-9ee6-3e53e86ea7fb)
